data/formats-data.js

```diff
@@ -4004,7 +4011,7 @@ exports.BattleFormatsData = {
 		tier: "Uber",
 	},
 	hippopotas: {
-		randomBattleMoves: ["earthquake", "slackoff", "whirlwind", "stealthrock", "protect", "toxic", "stockpile"],
+		randomBattleMoves: ["earthquake", "slackoff", "whirlwind", "stealthrock", "rockslide"],
```

Comment:
maybe rock tomb over rock slide

Reply:
why not both
its so weak i dont want tomb as the ONLY rock stab